msh changes

mhisltd opened the pull request:
Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
*
*
*

alectronic0 commented on testme.bat:
```diff
@@ -0,0 +1,17 @@
+gradlew.bat clean test --info
+#docker-compose -f docker-compose.yml -f docker-compose-test.yml run -e GRADLE_OPTS tests
+start build/reports/tests/test/index.html
+open build/reports/tests/test/index.html
```
can remove open & xdg as these are for unix and now windows
also tests should now be `document-management-inter.... (check the docker-compose-test.yml)

mhisltd replied:
Removed the requested lines and replaced the commented wget command with the Curl working command for windows.

alectronic0 commented on src/test/groovy/uk/gov/hmcts/dm/it/ReadDocumentIT.groovy:
```diff
@@ -239,35 +239,102 @@ class ReadDocumentIT extends BaseIT {
                 .when()
                 .get(documentUrl)
-
     }
```
If I am a user with no roles
I create a document with casework role + classification is private
then should I be able to get this file 

I need a test to prove this works please

mhisltd replied:
Added the requested test in the integration tests

alectronic0 commented:
I've updated your script try this on for size

mhisltd commented:
I've added some extra time to wait for the docker boxes to warmup.
- Fixed the issue of calling batches from testme.bat (They were initially not returning the control to the testme.bat after being executed.)
